source/_integrations/zwave_js.markdown

```diff
@@ -36,7 +36,9 @@ Please review the limitations [below](/integrations/zwave_js/#current-limitation
 
 ## Quick start (Home Assistant including Supervisor)
 
-To add Z-Wave JS to your installation, plug the Z-Wave stick into the device that runs Home Assistant. Then Go to Configuration >> Integrations in the UI. Click the "Add integration" button in the bottom right and from the list of integrations, select "Z-Wave JS" and follow the instructions shown.
+Z-Wave JS is compatible with all known Z-Wave Series 500 (Gen5) and Z-Wave Series 700 (Gen7) controller adapters in stick/dongle/module formats.
```

Comment:
This is not what the quickstart is about. Please remove this second. Users that want to read more about compatible hard, can read more about it below.